src/models/node.js

```diff
@@ -1179,7 +1179,7 @@ const Node = {
 
   getText() {
     return this.nodes.reduce((string, node) => {
-      return string + node.text
+      return node.kind == 'block' ? `${string}\n${node.text}` : string + node.text
```

Comment:
Even if you have only one block, it will always start with a link break.  I don't think we want that right?